optuna/integration/__init__.py

```diff
@@ -40,6 +41,8 @@
     from optuna.integration.chainermn import ChainerMNStudy  # NOQA
     from optuna.integration.cma import CmaEsSampler  # NOQA
     from optuna.integration.cma import PyCmaSampler  # NOQA
+    from optuna.integration.dask import DaskStorage  # NOQA
+    from optuna.integration.dask import OptunaSchedulerExtension  # NOQA
```

Comment:
`OptunaSchedulerExtension` seems like an internal class at first but does it have to be public? If not, we should probably avoid importing it here and prefix the class name with an underscore. Ditto for `register_with_scheduler`.

Reply:
Good point. Users shouldn't have to interact with `OptunaSchedulerExtension`, so I've removed it from the public API and prepended a `_` to the class name (along with a few other internal functions defined in `dask.py`)